### PR TITLE
chore: Remove disclaimer before GA

### PR DESCRIPTION
**Pull request by jjtang1985**
This PR should be merged just before the release.

### CODE REVIEW

**Review by ZhongpinWang — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,14 +5,8 @@
...
 Integrate chat completion into your business applications with SAP Cloud SDK for AI. Leverage the generative AI hub of SAP AI Core to make use of templating, grounding, data masking, content filtering and more. Setup your SAP AI Core instance with SAP Cloud SDK for AI Core.
 
-## Disclaimer ⚠️
-
-This project is currently in an experimental state. All functionality and naming is subject to change. Use at your own discretion.
-
 ## Table of Contents
 
-- [Disclaimer ⚠️](#disclaimer-️)
-- [Table of Contents](#table-of-contents)
```
Comment (deekshas8) [inline README.md:15]: [req] the table of contents should still be there

Comment (deekshas8) [reply README.md:15]: Ah I missed the ## before. Ignore my comment then

Comment (jjtang1985) [reply README.md:15]: It actually provided a link to itself 😄

File: README.md [hunk 2]
```diff
@@ -5,14 +5,8 @@
 
 Integrate chat completion into your business applications with SAP Cloud SDK for AI. Leverage the generative AI hub of SAP AI Core to make use of templating, grounding, data masking, content filtering and more. Setup your SAP AI Core instance with SAP Cloud SDK for AI Core.
 
-## Disclaimer ⚠️
-
-This project is currently in an experimental state. All functionality and naming is subject to change. Use at your own discretion.
-
 ## Table of Contents
```
Comment (ZhongpinWang) [inline README.md:8]:
```suggestion
### Table of Contents
```
At least for this famous VSCode extension `Markdown All in One`, if mark this section with three `#`, it won't auto generate a new entry for `Table of Contents` anymore.

**Review by marikaner — APPROVED** [review]

**Pull request merged**